docs/validators/sigstore_cosign.md

```diff
@@ -144,9 +144,22 @@ The secret can for example be created directly from your local `config.json` (fo
 ```bash
 kubectl create secret generic my-secret \
   --from-file=.dockerconfigjson=path/to/config.json \
-  --type=kubernetes.io/dockerconfigjson -n connaisseur
+  -n connaisseur
 ```
 
+The secret can also be generated directly from supplied credentials (which may differ from your local `config.json`, using:
+
+```bash
+kubectl create secret docker-registry my-secret \
+  --docker-server=https://index.docker.io/v1/ \
+  --docker-username='<your username>' \
+  --docker-password='<your password>' \
+  -n connaisseur
+```
+
+!!! note
+    At present, it [seems to be necessary](https://github.com/sigstore/cosign/issues/587#issuecomment-1062510930) to suffix your registry server URL with `/v1/`. This may become unnecessary in future. 
```

Comment:
```suggestion
    At present, it [seems to be necessary](https://github.com/sigstore/cosign/issues/587#issuecomment-1062510930) to suffix your registry server URL with `/v1/`. This may become unnecessary in the future. 
```
also: we try to do one sentence per line to improve editing  experience, as the newline will be interpreted as a single space when rendering